joomla/joomla-cms
# Fix for [4.0] Password Strength Bar Width is 100%.

Pull Request for Issue # .19227
Summary of Changes

Make Password Strength Bar Width equal to above input box
Testing Instructions

1/ Go to administrator/index.php?option=com_users
2/ Click New
3/ Enter a password
Expected result

Password Strength Bar Width is equal to above input box. This make nicer look.
